Makefile

```diff
@@ -43,6 +43,7 @@ PIP_FILE_LOCK = Pipfile.lock
 # default configuration
 ENV_FILE ?= .env.default
 HTTP_PORT ?= 5000
+ROUTE_PREFIX ?= /api/kml
```

Comment:
WHY not using SCRIPT_NAME allover?

Reply:
What do you mean exactly ? We cannot use the SCRIPT_NAME with the Flask development server, only with gunicorn.